modules/main.tf

```diff
@@ -100,7 +100,7 @@ locals {
       description        = "Helm chart deployment for apache airflow."
       project_root       = "modules/apache-airflow"
       space_id           = "root"
-      version_number     = "0.3.1"
+      version_number     = "0.3.5"
```

Comment:
I guess another candidate for parameterization could be the places where our version numbers are. If we had one place which holds all of the version numbers and then all of the references could then be pointed at that param, then you only need to increment it in one place.

Reply:
I'm not sure if I am totally on-board extracting it out to a single location since we might be in situations we want to test version A in a stack, but continue to run version B in another stack.

But I agree, there is work to improve this.